packages/dashboard/src/components/dashboard/getQueries.ts

```diff
@@ -4,9 +4,11 @@ import {
 } from '~/types';
 import { initialize } from '@iot-app-kit/source-iotsitewise';
 import { getClients } from './getClients';
+import { type EdgeMode } from '@iot-app-kit/core';
 
 export const getQueries = (
-  dashboardClientConfiguration: DashboardClientConfiguration
+  dashboardClientConfiguration: DashboardClientConfiguration,
```

Comment:
In `DashboardClientConfiguration`, `DashboardClientCredentials` contains `awsRegion` and in edge mode this would be set to `edge`. Would it be possible or make sense to just directly use `awsRegion === 'edge'` to determine if it was edgeMode instead of using an extra parameter?

Reply:
that's an interesting idea
Lemme see what we can do